vars/multicloudDeploy.groovy

```diff
@@ -22,25 +22,22 @@ import static com.sap.piper.Prerequisites.checkScript
 @Field Set STEP_CONFIG_KEYS = []
 
 @Field Set PARAMETER_KEYS = GENERAL_CONFIG_KEYS.plus([
-    /** The stage name. If the stage name is not provided, it will be taken from the environment variable 'STAGE_NAME'.*/
-    'stage',
     /** Defines the deployment type.*/
     'enableZeroDowntimeDeployment',
-    /** The source file to deploy to the SAP Cloud Platform.*/
+    /** Executes the deployments in parallel.*/
+    'parallelExecution',
+    /** The source file to deploy to SAP Cloud Platform.*/
     'source'
 ])
 
 /**
- * Deploys an application to multiple platforms (cloudFoundry, SAP Cloud Platform) or to multiple instances of multiple platforms or the same platform.
+ * Deploys an application to multiple platforms (Cloud Foundry, SAP Cloud Platform) or to multiple instances of multiple platforms or the same platform.
```

Comment:
typo, out of scope, but since it is a very small change I did add it in this PR.